src/main/java/net/library/model/response/UserResponse.java

```diff
@@ -0,0 +1,37 @@
+package net.library.model.response;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import net.library.model.entity.User;
+
+import java.util.List;
+
+public class UserResponse {
+
+    @JsonProperty("x-Correlation-id")
+    private String xCorrelationId;
```

Comment:
This field doesn't belong to this object. Could you move it in a general wrapper?